fix(challenge): update Statistics Calculator: Step 27 (#53196)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/65e62efde0592ec4b4bb6a69.md

---
id: 65e62efde0592ec4b4bb6a69
title: Step 31
challengeType: 0
dashedName: step-31
---

# --description--

> Now that you have a better understanding on how the `getMode` function works, you can remove the `console.log(getMode(numbers))` statement from the `calculate` function.

Reviewer: ```suggestion
Now that you have a better understanding of how the `getMode` function works, you can remove the `console.log(getMode(numbers))` statement from the `calculate` function.
```

---
id: 65e62efde0592ec4b4bb6a69
title: Step 31
challengeType: 0
dashedName: step-31
---

# --description--

> Now you understand how to use `forEach`, remove `console.log(getMode(numbers))` to proceed to the next section.

Reviewer 2: ```suggestion
Now that you have a better understanding on how the `getMode` function works, you can remove the `console.log(getMode(numbers))` statement from the `calculate` function.
```

# --hints--

> Remove `console.log(getMode(numbers))`

Reviewer 2: ```suggestion
You should not have a `console.log(getMode(numbers))` inside your `calculate` function.

```

---
id: 65e62efde0592ec4b4bb6a69
title: Step 31
challengeType: 0
dashedName: step-31
---

# --description--

Now that you have a better understanding on how the `getMode` function works, you can remove the `console.log(getMode(numbers))` statement from the `calculate` function.

# --hints--

You should not have a `console.log(getMode(numbers))` inside your `calculate` function.

```js
assert.notMatch(calculate.toString(), /console\.log\(\s*getMode\(numbers\)\s*\)/);

> 
> 

Reviewer 2: ```suggestion
```
